[+] Add a2581911655 (#206)
one-among-us/data: people/a2581911655/page.en.md

---

> name: jjlin1219

Reviewer: ```suggestion
name: Magic Chewing Gum
```

info:

>     alias: Kouxiangtang

Reviewer: ```suggestion
    alias: Chewing Gum, jjlin1219
```

    location: Urumqi, Xinjiang
---

## Description

> Kouxiangtang was a trans girl, she loved life and cared about others.

Reviewer: ```suggestion
Chewing Gum was a trans girl loving life and caring about others.
```

Author: 其实我在想使用Chewing Gum是否合适（x
当时我在音译和直译纠结了好久，可以说说直译的理由吗？
